lib/ui/form/form_screen.dart

```diff
@@ -105,6 +123,19 @@ class FormScreenState extends ConsumerState<FormScreen> {
                       question: questions[index - 1],
                       questionIndex: index,
                       questionTotal: questionTotal,
+                      onUpdatedAnswers: (answers) {
+                        List<SubmitSurveyAnswerModel> answerModels = [];
+                        final questionId = questions[index - 1].id;
+                        for (SubmitAnswerModel answer in answers) {
+                          answerModels.add(SubmitSurveyAnswerModel(
+                            id: answer.answerId,
+                            answer: answer.answerText ?? '',
+                          ));
+                        }
```

Comment:
Can we use the `map` operator to map the `answers` to `answerModels`? The above code looks too cumbersome.